proto/test/v1/conformance-test.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package google.api.expr.test.v1;
+
+import "google/api/expr/v1alpha1/checked.proto";
+import "google/api/expr/v1alpha1/eval.proto";
+import "google/api/expr/v1alpha1/value.proto";
+import "google/protobuf/empty.proto";
+
+// Configuration needed for running any test.
+message Config {
+  // Command line for executing the CelService server to be used
+  // when no call-specific server is specified.
+  string server_cmd = 1;
+
+  // Command line for executing a CelService server to be used
+  // for Parse() calls.  Default to server_cmd if empty.
+  string parse_server_cmd = 2;
+
+  // Command line for executing a CelService server to be used
+  // for Check() calls.  Default to server_cmd if empty.
+  string check_server_cmd = 3;
+
+  // Command line for executing a CelService server to be used
+  // for Eval() calls.  Default to server_cmd if empty.
+  string eval_server_cmd = 4;
+}
+
+message EvalResponseMatcher {
+  oneof kind {
+    google.api.expr.v1alpha1.Value value = 1;
+    ErrorSets errors = 2;  // expect one of these
+    UnknownSets unknowns = 3;  // expect one of these
+    string parse_failure_regex = 4;  // "." until we standardize
+    string check_failure_regex = 5;
+    google.protobuf.Empty trueval = 6; // shortcut for true boolean value
```

Comment:
I much prefer the missing value default to `true` for evaluation. Likewise, I would expect a missing `Check` output type to imply a type of `bool`.

Reply:
Implemented default eval matcher.  Disagree on the default check matcher - rather than requiring each test to stutter on non-boolean value, we can validate that the checker's inferred result type is compatible with the expected value in the eval matcher.  Added as a TODO for now.